lib/mutant/runner/sink.rb

```diff
@@ -1,107 +1,72 @@
 module Mutant
   class Runner
-    # Abstract base class for computation sinks
     class Sink
-      include AbstractType
+      include Concord.new(:env)
 
-      # sink status
+      # Initialize object
       #
-      # @return [Object]
+      # @return [undefined]
       #
       # @api private
-      abstract_method :status
+      def initialize(*)
+        super
+        @start           = Time.now
```

Comment:
@dkubb Not this iteration, out of focus. Its planned to reradicate all direct use of IO globals. But than I found global style fixes are needed before, than I found uncoverage, so this PR exists :P